Library/ViewModels/ProfileViewModel.swift

```diff
@@ -1,146 +0,0 @@
-import Foundation
-import KsApi
-import Prelude
-import ReactiveExtensions
-import ReactiveSwift
-
-public protocol ProfileViewModelInputs {
```

Comment:
Not sure if we wanna delete any files just yet since we are going to be toggling between versions using the flag?

Reply:
This `ProfileViewModel` and its' associated tests is not used anywhere in the app. I did a search for `ProfileViewModel` and `ProfileViewModelTests` in the app and it did not show anywhere in the entire project except in its own declaration.
I found this because I was looking for how `BackDashboardViewController` was implementing its' navigation bar, which is similar to the one we want in this page.

I believe its safe to delete, if you think its still being used somewhere let's keep it in? What do you think? I can make a separate PR for this as well :)